### ISSUE TITLE
mode for updating a single (sub-)album

### ISSUE THREAD

**Issue by dsschult** [saimn/sigal#508]
As background, I'm investigating speedups such that I can run sigal in an interactive mode whenever updates are made to a large gallery, instead of as a background task every hour.  Currently, running sigal takes several minutes, a large portion of which is reading and writing files that have not changed.

Since I know beforehand which files have changed (usually a single file or directory), it would be significantly faster to only process that one album and ignore all others.  Something like the inverse of the `ignore_directories` setting, except keeping the existing output for everything that is ignored.

Before trying to implement something like this myself, do you have any thoughts or preferences?

**Comment by saimn** [comment]
Yes indeed, something like `ignore_dirs` in the collection phase should be doable.
